src/task/mod.rs

```diff
@@ -10,9 +10,12 @@
 //! assert_eq!(ans, 42);
 //! ```
 
+#[cfg(not(feature = "once_cell"))]
+use std::cell::OnceCell as LazyCell;
```

Comment:
No, OnceCell and LazyCell are different.

Reply:
Do you mind to require crate user to choose `stable` feature explicitly?

I've added it and included 'dep:once_cell' as a dependency.

When `nightly` is enabled, we enable 'lazy_cell'

Reply:
Alternatively we could add `stable` to default features which seems as a safe default.